packages/cli/templates/app/.env

```diff
@@ -1,3 +0,0 @@
-# THIS FILE SHOULD NOT BE CHECKED INTO YOUR VERSION CONTROL SYSTEM
```

Comment:
I think we still want the `.env` file in the template version that we put into the generated app. We're not using Postgres by default anymore, but it's likely that any nontrivial production app will want to use postgres or something similar, so we can put a comment here telling the user how to use this file if that's the case. Something like:

```
# You can use this file to specify secrets like your database connection information
# if you move to a more sophisticated database than sqlite. For example, for postgres:
DATABASE_URL=postgresql://__username__@localhost:5432/__name__
```